### PR TITLE
docs: Review package scan instructions after refactoring

### PR DESCRIPTION
**Pull request by mcollovati**
Classpath scan restriction are now defined with annotation
instead of overriding scanPackage method.

Part of vaadin/testbench#1420

### CODE REVIEW

**Comment by mcollovati** [comment]
Draft until vaadin/testbench#1420 is closed

File: articles/tools/testbench/ui-unit-testing/getting-started.asciidoc [hunk 1]
```diff
@@ -35,17 +35,35 @@ class MyViewTest extends UIUnitTest {
 ----
 
 By default, the base class scans whole classpath for routes and error views, but this behavior can be changed for a faster bootstrap.
-To restrict the scan to a specific package and its sub-packages, override the [methodname]`scanPackage()` method and return the package name.
+To restrict the scan to specific packages and their sub-packages, annotate the test class with [annotationname]`ViewPackages` and specify the package by filling the [methodname]`classes()` array with classes that are member of the desired packages, or by providing the packages fully-qualified names in the [methodname]`packages()` property.
+Using [methodname]`classes()` is the preferred way, since it plays well with IDEs refactoring, when moving classes to different packages.
 
 [source,java]
```
Comment (caalador) [inline articles/tools/testbench/ui-unit-testing/getting-started.asciidoc:41]:
```suggestion
.Package scan examples
[source,java]
```

File: articles/tools/testbench/ui-unit-testing/getting-started.asciidoc [hunk 2]
```diff
@@ -35,17 +35,35 @@ class MyViewTest extends UIUnitTest {
...
+
+@ViewPackages(
+    classes={ MyView.class, OtherView.class },
+    packages={ "com.example.app.pgk1", "com.example.app.pgk2" }
+)
+class MyViewTest extends UIUnitTest {
+}
+----
+
+If the annotation is used without providing [methodname]`classes()` and [methodname]`packages()`, it acts a shortcut for using the test class package
```
Comment (caalador) [inline articles/tools/testbench/ui-unit-testing/getting-started.asciidoc:59]:
```suggestion
```

File: articles/tools/testbench/ui-unit-testing/getting-started.asciidoc [hunk 3]
```diff
@@ -35,17 +35,35 @@ class MyViewTest extends UIUnitTest {
...
 
-        @Override
-        protected String scanPackage() {
-                return MyView.class.getPackageName();
-        }
+Using the annotation without providing [methodname]`classes()` and [methodname]`packages()`, acts as a shortcut for restricting scan to current test class package and sub-packages.
```
Comment (caalador) [inline articles/tools/testbench/ui-unit-testing/getting-started.asciidoc:61]:
```suggestion
Using the annotation without providing [methodname]`classes()` or [methodname]`packages()`, acts as a shortcut for restricting scan to current test class package and sub-packages.
```

**Review by caalador — CHANGES_REQUESTED** [review]

File: articles/tools/testbench/ui-unit-testing/getting-started.asciidoc [hunk 4]
```diff
@@ -35,17 +35,35 @@ class MyViewTest extends UIUnitTest {
 ----
 
 By default, the base class scans whole classpath for routes and error views, but this behavior can be changed for a faster bootstrap.
-To restrict the scan to a specific package and its sub-packages, override the [methodname]`scanPackage()` method and return the package name.
+To restrict the scan to specific packages and their sub-packages, annotate the test class with [annotationname]`ViewPackages` and specify the package by filling the [methodname]`classes()` array with classes that are member of the desired packages, or by providing the packages fully-qualified names in the [methodname]`packages()` property.
```
Comment (dougjewsbury) [inline articles/tools/testbench/ui-unit-testing/getting-started.asciidoc:38]:
```suggestion
To restrict the scan to specific packages and their sub-packages, annotate the test class with [annotationname]`ViewPackages` and specify the package by filling the [methodname]`classes()` array with classes that are members of the desired packages, or by providing the packages with fully qualified names in the [methodname]`packages()` property.
```

File: articles/tools/testbench/ui-unit-testing/getting-started.asciidoc [hunk 5]
```diff
@@ -35,17 +35,35 @@ class MyViewTest extends UIUnitTest {
...
+class MyViewTest extends UIUnitTest {
+}
+----
+
 
-        @Override
-        protected String scanPackage() {
-                return MyView.class.getPackageName();
-        }
+Using the annotation without providing [methodname]`classes()` or [methodname]`packages()`, acts as a shortcut for restricting scan to current test class package and sub-packages.
```
Comment (dougjewsbury) [inline articles/tools/testbench/ui-unit-testing/getting-started.asciidoc:61]:
```suggestion
Using the annotation without providing [methodname]`classes()` or [methodname]`packages()` acts as a shortcut for restricting the scan to the current test class package and sub-packages.
```

**Review by dougjewsbury — COMMENTED** [review]
AsciiDoc looks good.

**Review by caalador — APPROVED** [review]

**Pull request merged**